### PR TITLE
Add clusterName to cleanup action

### PR DESCRIPTION
**Pull request by desistefanova**

### CODE REVIEW

File: mdb-realm/cleanup/action.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ inputs:
   privateapiKey:
     required: true
     description: The private API key.
+  clusterName:
+    required: true
```
Comment (nirinchev) [inline mdb-realm/cleanup/action.yml:21]: This should not be required - if it's not provided, we generate one.

Comment (desistefanova) [reply mdb-realm/cleanup/action.yml:21]: Yes, I saw that. But deleting a cluster with a newly generated name doesn't make sense to me. It allways fails. That's why I decided to be required. We have also DeleteAllCluster that could be used for all clusters.

Comment (nirinchev) [reply mdb-realm/cleanup/action.yml:21]: If you don't provide a name for the deploy action, we generate a deterministic name. In the same workflow, if you call the `cleanup` action, the generated name will be the same as the name we generate in the deploy action. So the cluster will exist - that's how we use those in the .NET workflow.

Comment (desistefanova) [reply mdb-realm/cleanup/action.yml:21]: I see. Ok I will make it not required.

Comment (desistefanova) [reply mdb-realm/cleanup/action.yml]: Done.

**Review by nirinchev — APPROVED** [review]

**Pull request merged**